### DISCUSSION TITLE
Enabling AWS Scans | Does it make sense?

### DISCUSSION THREAD

**Discussion by sherifkayad** [aquasecurity/trivy-operator#999] [Development]
Hello everyone, I am using the Trivy Operator and I am loving it!

Was just thinking about the best way to also scan our whole AWS account (s) .. and I don't really know if it makes sense / might be on the roadmap to enable the `aws` scans as part of the Trivy Operator.

I didn't want to open a feature request before really discussing if that makes sense in the first place or not.

What do you think? Feedback / discussion here is much appreciated.

**Comment by chen-keinan** [comment]
@sherifkayad are you familiar with `trivy aws..` scanning :
https://aquasecurity.github.io/trivy/v0.31.3/docs/cloud/aws/scanning/

**Reply by sherifkayad** [reply]
@chen-keinan thanks for your prompt reply. Yes, sure I am. I was wondering if adding the `aws` scanning to the Operator would also make sense or not. That's it

**Reply by chen-keinan** [reply]
@sherifkayad the operator is on the k8s cluster domain  , it live in side the cluster and scan it. You could scan individual aws clusters.

scanning cloud via aws account is more `trivy` use case

**Reply by sherifkayad** [reply]
Got ya thanks 👍
